belvo/client.py

```diff
@@ -1,17 +1,19 @@
-import os
-
 from . import resources
 from .exceptions import BelvoAPIException
 from .http import APISession
+from .utils import url_resolver
 
 
 class Client:
-    def __init__(self, secret_key_id: str, secret_key_password: str, url: str = None) -> None:
-        if url is None:
-            url = os.getenv("BELVO_API_URL")
+    def __init__(self, secret_key_id: str, secret_key_password: str, url: str = "") -> None:
```

Comment:
there's no need to change the constructor here. It does the same things, because `URL_ENVIRONMENTS.get(None) = None`. it's same things though

Reply:
since the user can pass the environment by **name** ("sandbox") or by **url** (https://sandbox.belvo.com), can we rename `url` param to `environment`? 
@menecio would this break  any customer implementation? AFAIK the Client is initiated like this:
`Client("Secret Key ID", "Secret Key PASSWORD", "https://sandbox.belvo.com")`. 
But can it also be initiated like this? If yes, it can break sth & we can ignore this.
`Client("Secret Key ID", "Secret Key PASSWORD", url="https://sandbox.belvo.com")`
```suggestion
    def __init__(self, secret_key_id: str, secret_key_password: str, environment: str = "") -> None:
```

Reply:
I think we should consider some users use `url="https://..."`. BTW, i found the solution for my previos comment about linting, 
```
def url_resolver(url: Optional[str]):
    ....
```

Reply:
then you can dismiss my previous comment about renaming `url` to `environment`.
great Beni 💪🏼